core/src/main/resources/reference.conf

```diff
@@ -1611,6 +1611,15 @@ datastax-java-driver {
         significant-digits = 3
         refresh-interval = 5 minutes
       }
+
+      # The time after the node level metrics will be evicted.
+      # The eviction will happen only if none of the enabled node-level metrics is updated for a given node for a given time.
+      # When the expire-after will elapse, all metrics for a given node are removed.
+      #
```

Comment:
Also specify that it will be forced to 1 hour (currently, see my other comment) if you set a lower value.